### DISCUSSION TITLE
How to move a floating window to the top of the screen?

### DISCUSSION THREAD

**Discussion by mcnuggetsx20** [qtile/qtile#3846] [Q&A]
I have a window which I want to be in floating state and at the very top of my screen. When I call ```cmd_set_position_floating(0,0)``` it goes right under the top bar instead of the top of the screen. Is there a way to put it on top? I have tried ```cmd_set_position(0,0)``` as well.

**Comment by m-col** [comment]
What bar are you using? I suspect the bar is trying to reserve space, which Qtile is informed of so it can adjust the tiling layout, for example, and it might be also using the reserved space to adjust the position of this window. By that I mean, that the (0, 0) coordinate you're requestion would be interpreted as being relative to the part of the screen that isn't reserved. That said, from a quick look in the code I don't think it actually does that!

Anyway, a solution might be to just pass a negative Y coordinate: `cmd_set_position_floating(0,-24)`
